ui/apps/platform/src/Containers/Collections/types.ts

```diff
@@ -72,6 +72,8 @@ export function isByLabelSelector(
     return isByLabelField(selector.field);
 }
 
+export type CollectionSlim = { id: string; name: string; description: string };
```

Comment:
Note to self - we can just use the full type since we are querying the server for the full `Collection` object here.
- [ ] Replace `CollectionSlim` with `Collection`

Reply:
Instead of `Collection`, I just passed `CollectionResponse[]` directly since the parsing of the response into a stricter data type is not needed for this piece of the UI.